ios/engine/KMEI/KeymanEngineTests/MigrationTests.swift

```diff
@@ -30,7 +30,7 @@ class MigrationTests: XCTestCase {
 
     sil_euro_latin_kbds.forEach {
       XCTAssertEqual($0.packageID, "sil_euro_latin")
-      XCTAssertEqual($0.version, TestUtils.Keyboards.sil_euro_latin.version)
+      XCTAssertEqual($0.version, Defaults.keyboard.version)
```

Comment:
A test for updating to the current defaults should _probably_ compare against the default's version, rather than any not-updated test-version of that resource.